client/.env.development

```diff
@@ -1,2 +1,2 @@
-VITE_SERVER_HOST=http://localhost:4000
-VITE_SPOTIFY_CLIENT_ID=f29edb16e8cc4dd1ba897ad146750fa2
\ No newline at end of file
+VITE_SERVER_HOST=https://spotify-showcase-production-d157.up.railway.app/
```

Comment:
@michael-watson just to double check, this is the URL you talked about using correct?